### PR TITLE
secure channel log should use secure channel tag

### PR DESCRIPTION
**Pull request by kghost**
#### Problem
Log tags in secure_channel are wrong, probably caused by refactoring and moving code around.

#### Change overview
Use the correct log tag

#### Testing
This is a straightforward change, no extra test is needed.

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

**Review by tcarmelveilleux — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

**Review by woody-apple — CHANGES_REQUESTED** [review]
Per the updated template, can you update the PR here?

```
#### Problem
What is being fixed?  Examples:
* Fix crash on startup
* Fixes #12345 12345 Frobnozzle is leaky (exactly like that, so GitHub will auto-close the issue).

#### Change overview
What's in this PR

#### Testing
How was this tested? (at least one bullet point required)
    • If unit tests were added, how do they cover this issue?
    • If unit tests existed, how were they fixed/modified to prevent this in future?
    • If integration tests were added, how do they verify this change?
    • If manually tested, what platforms controller and device platforms were manually tested, and how?
    • If no testing is required, why not?
```

**Review by LuDuda — APPROVED** [review]

**Comment by andy31415** [comment]
@kghost regarding testing "This is a straightforward change, no extra test is needed." ... At least smoke test is required. I agree formal CI test is not required, however please run manually a smoke test and document it here. Like

`Ran .... and saw .... in the logs`. In your case this should prove that SecureChannel is indeed used in the logs as expected.

TLDR: "did not run tests beside compilation" is almost never acceptable. You should at least manually run an app to validate it starts up. You can rely on CI that it starts up but then you have to validate that CI logs show the change you expect.
